src/controllers/users.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
+
+  if (!foundedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundedUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
+
+  if (!foundedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  userService.remove(userId);
+  res.sendStatus(204);
+};
+
+const update = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
+
+  if (!foundedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const { name } = req.body;
+
+  if (!name || typeof name !== 'string') {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  userService.update({
+    userId,
+    name,
+  });
+  res.send(foundedUser);
```

Comment:
After updating the user information, you should send the updated user data back in the response. Currently, the original user data before updating is being sent.